with open("files/file.txt","r") as f:
    f.read()
    raise ValueError("xato")
